Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:latest
+
+WORKDIR /user/src/uems/front
+
+EXPOSE 15300
+
+COPY package*.json ./
+
+RUN npm install
+
+RUN npm install -g serve
```

Comment:
Will open a new pull request with the new system when it is ready as discussed but this is all cool for now

Reply:
Yeah awesome